dom0/sd-base-template-files.sls

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# vim: set syntax=yaml ts=2 sw=2 sts=2 et :
+include:
+  - fpf-apt-repo
+
+# TODO: test if this works, if not it may need to be a qvm-run command
+sd-base-template-install-additional-packages:
+  pkg.installed:
+    - pkgs:
+      - qubes-core-agent-passwordless-root
```

Comment:
All of these should be dependencies in -config or other packages. But I'm not sure why we're not starting with the GNOME (or even XFCE) templates to begin with, I thought that was what we had discussed initially?

Reply:
If we start with -gnome or -xfce, then the base template will include more than we want. The initial discussion was actually around using -minimal, I did flip to -xfce temporarily, but the resulting attack surface in terms of unused packages is too great, and removing them would be more hassle than starting with minimal and adding just what we need.

Reply:
But I'm down to add them in the securedrop-workstation-config package.